### ISSUE TITLE
[feature request] Support for more languages

### ISSUE THREAD

**Issue by Naesstrom** [Librarika/Issues#260]
Might alreade be in the plans but I couldn't find any information regarding it.
Please make it possible to translate Librarika and upload a translation.

**Comment by rayhan** [comment]
We are going to start translating Librarika soon. Any help in this regard will be extremely helpful.

**Comment by Naesstrom** [comment]
Lovely, waiting for some more information on how to translate!

**Comment by andrewalker** [comment]
I am also interested in this! Particularly pt-BR.

**Comment by andrewalker** [comment]
Is there a way to help?

**Comment by rayhan** [comment]
Yes, I have finally found a good tool for language translation collaboration. 

How to contribute?
Please create an account at Zanata [https://translate.zanata.org] and email me your username. I will then add you as a collaborator.

Translation URL?
https://translate.zanata.org/iteration/view/librarika/v1

We appreciate any help regarding translations.

**Comment by ckorobov** [comment]
Guys, any news with UI translations? I would like to contribute if possible in adding more languages

**Comment by WebShapedBiz** [comment]
Hi.
I've just created a Zanata account to help translate Librarika. My username is `sinisaperovic` and I'd like you to add me as a collaborator.

Cheers.

**Comment by S-mardii** [comment]
Hello. I am exploring the translation option for the Library into my local language. There is a guide on translating via Zanata.

However, when signing up, I received error message about the maintenance of the upstream Zatana project.

Please find more information here: https://translate.zanata.org/?dswid=-694

Are you considering using other translation tools like Transifix?
